doc/changes/latest.inc

```diff
@@ -49,6 +49,8 @@ Enhancements
 
 - Add support for interpolating oxy and deoxyhaemoglobin data types (:gh:`9431` by `Robert Luke`_)
 
+- Add support for multiple datablocks (acquistions with pauses) in `mne.io.read_raw_nihon` (:gh:`9437` by `Fede Raimondo`_)
```

Comment:
```suggestion
- Add support for multiple datablocks (acquistions with pauses) in :func:`mne.io.read_raw_nihon` (:gh:`9437` by `Fede Raimondo`_)
```

Reply:
It's already adding the two events exactly as you say: `BAD boundary` and `EDGE boundary`.